libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/model/DBTableIndex.java

```diff
@@ -26,7 +26,8 @@
 
 @Data
 @EqualsAndHashCode(
-        exclude = {"name", "warning", "schemaName", "tableName", "createTime", "updateTime", "ordinalPosition"})
```

Comment:
Because `DBTableIndexEditor` uses `Objects.equals(oldIndex, newIndex)` when judging whether two indexes are equal. I think `owner` is just like `schemaName` should not be included when judging whether two indexes are equal.